docs/mrdocs.schema.json

```diff
@@ -0,0 +1,255 @@
+{
+  "$schema": "https://json-schema.org/draft-07/schema",
```

Comment:
https://json-schema.org/draft/2020-12/schema ?

Reply:
Yea, we could update to the new draft. Most schemas I saw were using older drafts, but I don't think that should be a problem.